etna/core/models/basepage.py

```diff
@@ -49,12 +49,37 @@ class BasePage(MetadataPageMixin, DataLayerMixin, Page):
         blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
+        help_text=_("Image that will appear on thumbnails and promos around the site."),
     )
 
     # DataLayerMixin overrides
     gtm_content_group = "Page"
 
-    promote_panels = MetadataPageMixin.promote_panels + [
+    # Overriding the original help_text set by default in MetadataPageMixin
+    promote_panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel(
+                    "slug",
+                    help_text=_(
+                        "The name of the page as it will appear at the end of the URL e.g. http://nationalarchives.org.uk/[slug]"
+                    ),
+                ),
+                FieldPanel("seo_title"),
+                FieldPanel("show_in_menus"),
+                FieldPanel(
+                    "search_description",
+                    help_text=_(
+                        "The descriptive text displayed underneath a headline in search engine results and when shared on social media."
```

Comment:
Note: is `meta_description` the same as search_description? I'm not sure if the ticket is wrong?

Reply:
Meta description is `search_description`, yeah - the label comes up as "Meta description" in the admin